src/platform/utilities/sso/index.js

```diff
@@ -13,16 +15,20 @@ function keepAlive() {
   return environment.isLocalhost() ? mockKeepAlive() : liveKeepAlive();
 }
 
-async function hasVAGovSession() {
+async function vaGovProfile() {
   try {
     const resp = await fetch(`${environment.API_URL}/v0/user`, {
-      method: 'HEAD',
+      method: 'GET',
       credentials: 'include',
     });
-    return resp.ok;
+    if (resp.ok) {
+      const json = await resp.json();
+      return camelCaseKeysRecursive(json.data.attributes.profile);
```

Comment:
@patrickvinograd so do you thinking wrapping this in the `camelCaseKeysRecursive` function is unnecessary?

Reply:
well...the olive_branch inflection thing is *not* recursive - so it will only permute the top-level keys. So...I don't know, in this case? But just wanted to make it clear that there is some transformation happening.